src/lib/expression/between_expression.hpp

```diff
@@ -6,15 +6,26 @@
 
 namespace opossum {
 
-class BetweenExpression : public AbstractPredicateExpression {
+class BaseBetweenExpression : public AbstractPredicateExpression {
  public:
-  BetweenExpression(const std::shared_ptr<AbstractExpression>& value,
-                    const std::shared_ptr<AbstractExpression>& lower_bound,
-                    const std::shared_ptr<AbstractExpression>& upper_bound);
+  static PredicateCondition get_between_predicate_expression(bool left_inclusive, bool right_inclusive);
```

Comment:
these four static functions don't have too much to do with the class they are in and should be - just like `is_binary_predicate_condition` - free functions. If they are only used in a single cpp, put them there instead. Again, this is to keep the interfaces of the expression classes noise-free